guides/common/modules/proc_creating_an_application_instance.adoc

```diff
@@ -0,0 +1,26 @@
+[id="{context}_creating_an_application_instance"]
+= Creating an Application Instance
+
+You can create xref:{context}_application_instances[application instances] as a blueprint for users to provision and configure hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Application > App Instances* and click the *New Application Instance* button.
+. In the *Name* field, enter the name of the new application instance.
+. In the *Description* field, enter an arbitrary description.
+. From the *Application Definition* menu, select an existing application definition.
+. In the table, connect the hosts and necessary services:
+** In the *Hostname* field, enter a unique name for the host.
+You may choose from all lowercase characters, digits, and hyphens.
+Hosts may not start with a hyphen.
```

Comment:
```suggestion
Hosts cannot start with a hyphen.
```
From IBM style guide: 

"“May” is overused in technical information and should in most instances be changed to “can” or “might.”"

Reply:
Thanks, applied!